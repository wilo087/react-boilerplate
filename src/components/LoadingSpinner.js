import React from 'react';

const LoadingSpinner = () => (
  <div className="loader-container">
    <div className="loader" />
  </div>
);

export default LoadingSpinner;
