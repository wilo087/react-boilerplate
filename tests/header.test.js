/* eslint-disable */
import React from 'react';

import renderer from 'react-test-renderer';
import LoadingSpinner from '../src/components/LoadingSpinner';


it('Header is rendering correctly', () => {
  const tree = renderer
    .create(<LoadingSpinner />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
