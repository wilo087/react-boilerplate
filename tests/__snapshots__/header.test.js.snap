// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header is rendering correctly 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  />
</div>
`;
